lib/addons/src/typings.d.ts

```diff
@@ -1,3 +1,20 @@
 declare module 'global';
-declare module '@storybook/client-logger';
-declare module '@storybook/addons';
\ No newline at end of file
+
+declare global {
+  interface NodeModule {
```

Comment:
i think some TS config is gone rogue here. `__WebpackModuleApi.Module` (from which `NodeModule` extends when using types `webpack-env`) already declares `hot`

see https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/webpack-env/index.d.ts

Reply:
yeah that makes more sense to use indeed